frontend/dispatcher.lua

```diff
@@ -417,16 +417,16 @@ function Dispatcher:addItem(caller, menu, location, settings, section)
                     end,
                     callback = function(touchmenu_instance)
                         local SpinWidget = require("ui/widget/spinwidget")
-                        local requested_precision = settingsList[k].precision
-                        if settingsList[k].step and math.floor(settingsList[k].step) ~= settingsList[k].step then
-                            requested_precision = "%0.1f"
+                        local precision
+                        if settingsList[k].step and settingsList[k].step < 1 then
+                            precision = "%0.1f"
                         end
```

Comment:
Leaving the math.floor(....) would ensure, that the precision is set to dezimal. This could be necessary if someone wants "1.5" steps. Who knows?

Reply:
will revert before i finish this thing